Doc/library/stdtypes.rst

```diff
@@ -319,7 +319,9 @@ the operations, see :ref:`operator-summary`):
 Notes:
 
 (1)
-   Also referred to as integer division.  The resultant value is a whole
+   Also referred to as integer division.  For operands of type :class:`int`,
+   the result has type :class:`int`.  For operands of type :class:`float`,
```

Comment:
I think, some number of readers can understand this as "If both operands are float, the result has type float".
But, in fact, if one of operands are float, the result type is float.

Reply:
> But, in fact, if one of operands are float, the result type is float.

This is already covered earlier in the docs where it talks about mixed arithmetic. There isn't room for an exhaustive combinatorial description here.

Similarly, let's leave details about `Fraction` and `Decimal` to the respective module documentation. All we need to cover here is `int`, `float` and `complex`.

Reply:
> This is already covered earlier in the docs where it talks about mixed arithmetic.

Indeed, right above the table ("when a binary arithmetic operator has operands of different numeric types, the operand with the “narrower” type is widened to that of the other...").